tests/test1.c

```diff
@@ -307,6 +308,25 @@ int main(int argc, char **argv)
 	}
 	printf("my_object.to_string()=%s\n", json_object_to_json_string(my_object));
 
+	json_object_put(my_array);
```

Comment:
I think you end up with a double free here, or rather down below on line 334.

Reply:
Thanks for pointing out, fixed.